tasmota/language/fr_FR.h

```diff
@@ -263,13 +263,13 @@
 #define D_RESTART_IN "Redémarrage dans"
 #define D_SECONDS "secondes"
 #define D_DEVICE_WILL_RESTART "Le module va redémarrer dans quelques secondes"
-#define D_BUTTON_TOGGLE "on/off"
+#define D_BUTTON_TOGGLE "on/off"     // Not better in french
```

Comment:
M/A

Reply:
Agree. A headache for me , because if I recall it's used in the main GUI ! In some forks I even use icons for the UP/DN of my roller-shutters.